### DISCUSSION TITLE
Modify crendentials of Tidar account

### DISCUSSION THREAD

**Discussion by fefepaille** [RandomNinjaAtk/docker-lidarr-extended#184] [Q&A] [answered]
Hello, 
First of all, thank a lot for your work, it's amazing ! Running on Unraid perfectly !

I would like to know if there is a way to change the binded Tidar account. 
I looked for it in vain. 

Thanks

**Comment by RandomNinjaAtk** [comment, marked as answer]
If you delete the files in `/config/xdg`, specifically `.tidal-dl.token.json`, that should require you to re-authenticate.

**Reply by fefepaille** [reply]
That was the only folder I didn't explored...
Thanks a lot, it worked !
